GPU/GPUTracking/TRDTracking/GPUTRDTracker.cxx

```diff
@@ -12,8 +12,8 @@
 /// \author Ole Schmidt
 
 //#define ENABLE_GPUTRDDEBUG
-#define ENABLE_WARNING 0
-#define ENABLE_INFO 0
+#define ENABLE_WARNING 1
+#define ENABLE_INFO 1
```

Comment:
This is reverted in the next commit.
No big deal, but for the future I'd try to avoid committing temporary debug changes :)